lib/modules/versioning/node/readme.md

```diff
@@ -1,3 +1,5 @@
-Renovate's Node.js versioning is a wrapper around npm's versioning, except that it makes sure to strip "v" prefixes from exact versions when replacing.
+Renovate's Node.js versioning is a wrapper around npm's versioning.
+But Renovate removes any `v` prefixes from exact versions when replacing.
```

Comment:
It's not clear to me what we mean with `exact` versions here:

- Exact version: `v1.0.0`?
- Loose version: `v1`?
- Exact is the same as pinned: `v1.0.0`?
- Something else

Also _when replacing_, is vague to me as well.

Reply:
I checked all of the scenarios and it didn't matter what the current version was. The `v` prefix is stripped regardless.

Also "replacing" here appears to refer to the `RangeStrategy` used:

https://github.com/renovatebot/renovate/blob/f1f5f270f08711e71e370118ee0e3e642af9a34d/lib/types/versioning.ts#L6

I think it's an abstraction leak from code to documentation. But again, the actual `RangeStrategy` doesn't appear to impact whether or not we remove the `v` prefix.

Reply:
### Response to comment

> I checked all of the scenarios and it didn't matter what the current version was. The `v` prefix is stripped regardless.

Okay! So we can just say that Renovate always removes the `v` prefix.

> I think it's an abstraction leak from code to documentation.

I don't understand what you're saying here. I think you mean that "translating code to human docs text" went wrong somewhere?

> But again, the actual `RangeStrategy` doesn't appear to impact whether or not we remove the `v` prefix.

Thanks for explaining that. I got confused because the `rangeStrategy` docs mention `pin` and `replace` behaviors. 😄 

### Suggestion to review/edit

```suggestion
If the version has a `v` prefix, Renovate always removes it when creating the update.
```